tgstation.dme

```diff
@@ -6,6 +6,51 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "html"

```

Comment:
Help --> preferences for tgstation --> Untick "Automatically set FILE_DIR", commit the dme, push the new commit.